### ISSUE TITLE
Use `Single Reserve` as `SETT` and `Multi Standard` as SettCurrencies

### ISSUE THREAD

**Issue by balqaasem** [setheum/setheum#189]
1. `ReserveCurrencyId`,
2. `StandardCurrencyIds`

For example (in NewRome runtime)
```
pub ReserveCurrencyId: SETT;
pub StandardCurrencyIds: Vec<CurrencyId> = vec![rUSD, rCHF, rEUR];
```

**Comment by balqaasem** [comment]
Works upon #177. Closes #177

**Comment by balqaasem** [comment]
Works upon and closes #151

**Comment by balqaasem** [comment]
CLOSES ON #168
